android/src/main/kotlin/com/airship/flutter/EventManager.kt

```diff
@@ -45,7 +46,7 @@ internal class AirshipEventStream(eventType: EventType) : EventChannel.StreamHan
     fun notifyEvent(event: Event) {
         var sink = eventSink
         if (sink != null) {
-            sink.success(event.eventBody?.toString())
+            sink.success(event.eventBody?.getString(event.eventBody.toString()))
```

Comment:
That looks really weird. What is happening here?

Reply:
getString is a json parsing method that requires a fallback string. The previous implementation was just passing toString which isn't properly parsed and comes out as ""contents"" instead of "contents". I wasn't sure what to put as the default in this case. We could also put an empty string or something. I figured we'd fall back to the guaranteed toString output in the event a non-json encoded string body is provided.

Reply:
OK.

Reply:
I think you could just do optString right?